tests/test_fbs_target.py

```diff
@@ -69,17 +71,24 @@ def test_get_script_config(self):
         )
 
         script_config_expected = {
-            "targetid": target.targetid,
-            "band_filter": target.filter,
+            "targetid": int(target.targetid),
+            "band_filter": str(target.filter),
+            "filter_name": (
+                f"RUBIN{target.filter}_random_filter_name"
+                if "band" in observation.dtype.names
+                else target.filter
+            ),
             "name": target.get_target_name(),
+            "note": str(target.note),
             "ra": target.get_ra(),
             "dec": target.get_dec(),
             "alt": target.alt,
             "az": target.az,
-            "rot": target.rot,
-            "rot_sky": target.ang,
+            "rot": float(target.rot),
+            "rot_sky": float(target.ang),
```

Comment:
Interesting that explicit casts are necessary for these two values and only for these two. Any idea why?